src/scripts/main.js

```diff
@@ -1,3 +1,17 @@
 'use strict';
 
-// write code here
+const mainList = document.querySelector('.tree');
+const listName = document.querySelectorAll('li');
+
+[...listName].forEach(elem => {
+  const span = document.createElement('span');
+
+  elem.prepend(span);
+  span.append(span.nextSibling);
+});
+
+mainList.addEventListener('click', e => {
+  const item = e.target.closest('span').nextSibling;
```

Comment:
Check if 'e.target.closest('span')' is not null before calling 'nextSibling' on it to avoid possible errors.

```suggestion
  const item = e.target.closest('span')?.nextSibling;
```